crates/viewer/re_redap_browser/src/collections.rs

```diff
@@ -34,21 +34,20 @@ pub struct Collections {
 }
 
 impl Collections {
-    pub fn add(
+    pub fn add_or_refresh(
```

Comment:
Nit: It now unconditionally re-fetches, so maybe just `fetch`?
```suggestion
    pub fn fetch(
```